auditlog/data/ir_cron.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+
+        <record id="ir_cron_auditlog_autovacuum" model="ir.cron">
+            <field name='name'>AutoVacuum audit logs</field>
```

Comment:
I just copied https://github.com/OCA/connector/blob/9.0/connector/queue/queue_data.xml#L31 to be identical with others and have a nice and coherent overview of the scheduled actions in the tree view:
- AutoVacuum audit logs
- AutoVacuum osv_memory objects
- AutoVacuum Queue Jobs